### ISSUE TITLE
@Final field field_187234_c:Ljava/util/Map; in performant.mixins.json:entity.EntityDataManagerMixin should be final

### ISSUE THREAD

**Issue by SonicSpeedster16** [someaddons/performant_issues#106]
**Describe the bug. 1.12 is no longer supported, do not make bugreports for it.**
I noticed the following warning in my server log and thought I would report it in case it is a real issue. Note that it seems to work fine on the server despite the warning:

`[modloading-worker-3/WARN] [mixin/]: @Final field field_187234_c:Ljava/util/Map; in performant.mixins.json:entity.EntityDataManagerMixin should be final`

**To Reproduce**
1. Start dedicated server
2. View latest.log

**Expected behavior**
No warnings in the log?

** Mod version **
Minecraft 1.16.5
Forge 36.0.14
Performant 3.30m

** Logs **
latest.log: https://gist.github.com/SonicSpeedster16/af359b4c3a577656aa0320829077b343

**Comment by yuuki-san** [comment]
may be relevant:
```
...
[HH:MM:SS] [main/WARN] [mixin/]: @Mixin target net/minecraft/world/server/ChunkManager$EntityTracker is public in performant.mixins.json:world.chunk.ChunkManagerEntityTrackMixin and should be specified in value
...

[HH:MM:SS] [modloading-worker-11/WARN] [mixin/]: @Final field field_187234_c:Ljava/util/Map; in performant.mixins.json:entity.EntityDataManagerMixin should be final
...
```

**Comment by someaddons** [comment]
should be fixed in 3.40m
